tests/sub_port_interfaces/conftest.py

```diff
@@ -369,6 +373,12 @@ def apply_route_config_for_port(request, duthost, ptfhost, define_sub_ports_conf
             # Remove L3 RIF from the PTF
             remove_ip_from_ptf_port(ptfhost, src_port, next_hop_sub_ports['ip'])
```

Comment:
Where are we adding back that port that got removed from vlan1000 in L306?

Reply:
Here:
https://github.com/Azure/sonic-mgmt/blob/1c85de1bbe302748d822c7ca22f0b4fcf39db004/tests/sub_port_interfaces/conftest.py#L400